src/components/RatingModal/index.js

```diff
@@ -0,0 +1,52 @@
+import { Modal, Button } from 'react-bootstrap'
```

Comment:
you can replace `Modal` with the `Modal` component (from `material-ui`) that was recently merged into `master` branch. We aim to migrate from `react-bootstrap` to `material-ui`

Reply:
I take care of it and thank you for the different comments.